### DISCUSSION TITLE
how to use the three argument variant of the button command to link three buttons.

### DISCUSSION THREAD

**Discussion by Terracotta-62** [artisan-roaster-scope/artisan#1231] [General]
Examples  Playback Events  by BT ,by time ,Playback Stop.

Thanks!

**Comment by myTonino** [comment]
Which "three argument variant of the button command" do you refer here? Custom event buttons can have a state but it is binary (see [Buttons – Appearance, State, and Action](https://artisan-roasterscope.blogspot.com/2023/06/buttons-appearance-state-and-action.html)).

You could define a sequence of 3 buttons which use a technic similar to example 2 of the post linked above to ensure that only one of the 3 buttons is active.
